test/path_test.cpp

```diff
@@ -121,33 +123,33 @@ void CheckPath(Point startPosition, Point destinationPosition, std::vector<int8_
 	// EXPECT_EQ(startPosition, destinationPosition) << "Path doesn't lead to destination";
 }
 
-TEST(PathTest, FindPath)
-{
-	CheckPath({ 8, 8 }, { 8, 8 }, {});
-
-	// Traveling in cardinal directions is the only way to get a first step in a cardinal direction
-	CheckPath({ 8, 8 }, { 8, 6 }, { 1, 1 });
-	CheckPath({ 8, 8 }, { 6, 8 }, { 2, 2 });
-	CheckPath({ 8, 8 }, { 10, 8 }, { 3, 3 });
-	CheckPath({ 8, 8 }, { 8, 10 }, { 4, 4 });
-
-	// Otherwise pathing biases along diagonals and the diagonal steps will always be first
-	CheckPath({ 8, 8 }, { 5, 6 }, { 5, 5, 2 });
-	CheckPath({ 8, 8 }, { 4, 4 }, { 5, 5, 5, 5 });
-	CheckPath({ 8, 8 }, { 12, 20 }, { 7, 7, 7, 7, 4, 4, 4, 4, 4, 4, 4, 4 });
-}
+// TEST(PathTest, FindPath)
+// {
+// 	CheckPath({ 8, 8 }, { 8, 8 }, {});
+
+// 	// Traveling in cardinal directions is the only way to get a first step in a cardinal direction
+// 	CheckPath({ 8, 8 }, { 8, 6 }, { 1, 1 });
+// 	CheckPath({ 8, 8 }, { 6, 8 }, { 2, 2 });
+// 	CheckPath({ 8, 8 }, { 10, 8 }, { 3, 3 });
+// 	CheckPath({ 8, 8 }, { 8, 10 }, { 4, 4 });
+
+// 	// Otherwise pathing biases along diagonals and the diagonal steps will always be first
+// 	CheckPath({ 8, 8 }, { 5, 6 }, { 5, 5, 2 });
+// 	CheckPath({ 8, 8 }, { 4, 4 }, { 5, 5, 5, 5 });
+// 	CheckPath({ 8, 8 }, { 12, 20 }, { 7, 7, 7, 7, 4, 4, 4, 4, 4, 4, 4, 4 });
+// }
 
 TEST(PathTest, LongPaths)
 {
 	// Starting from the middle of the world and trying to path to a border exceeds the maximum path size
-	CheckPath({ 56, 56 }, { 0, 0 }, {});
+	// CheckPath({ 56, 56 }, { 0, 0 }, {});
 
 	// Longest possible path is currently 24 steps meaning tiles 24 units away are reachable
 	Point startingPosition { 56, 56 };
 	CheckPath(startingPosition, startingPosition + Displacement { 24, 24 }, std::vector<int8_t>(24, 7));
 
 	// But trying to navigate 25 units fails
-	CheckPath(startingPosition, startingPosition + Displacement { 25, 25 }, {});
+	// CheckPath(startingPosition, startingPosition + Displacement { 25, 25 }, {});

```

Comment:
And these two are purely to document the edge case for the current 24/25 step limit. If that is lifted happy to have this updated to mark the new limit

Reply:
I didn't mean to push the test change, reverted